### PR TITLE
missingSrc in Ant

### PR DESCRIPTION
**Pull request by fabvador**

### CODE REVIEW

File: src/main/java/org/vafer/jdeb/ant/Data.java [hunk 1]
```diff
@@ -29,6 +29,8 @@
 import org.vafer.jdeb.producers.DataProducerDirectory;
 import org.vafer.jdeb.producers.DataProducerFile;
 
+import static org.vafer.jdeb.ant.MissingSourceBehavior.*;
```
Comment (tcurdt) [inline src/main/java/org/vafer/jdeb/ant/Data.java]: No * imports please.

**Comment by tcurdt** [comment]
Looking OK except for the * import. Thanks!

**Comment by tcurdt** [comment]
If you could fix that I am happy to merge this.

**Comment by fabvador** [comment]
ok no probs but this is a copy/paste of what is in maven.
why should it be different?r
I was actually wondering why it is so in maven.data!

**Comment by tcurdt** [comment]
Urhg. OK.

Then I'll accept the PR as is and we need to fix both.

Thanks, @fabvador!

**Pull request merged**